packages/ra-ui-materialui/src/field/ReferenceField.tsx

```diff
@@ -146,25 +145,21 @@ export const ReferenceFieldView = <
     if (link) {
         return (
             <Root className={className} sx={sx}>
-                <RecordContextProvider value={referenceRecord}>
```

Comment:
The RecordContextProvider was added twice, as `ReferenceFieldBase` already does it.